test/functional/pdo_sqlsrv/pdostatement_Buffqry.phpt

```diff
@@ -1,200 +1,206 @@
-	$query = "UPDATE PDO_AllTypes SET [BitCol]=:Name WHERE [BigIntCol]=:value";
-	$stmt = $db->prepare($query,array(PDO::ATTR_CURSOR=>PDO::CURSOR_SCROLL, PDO::SQLSRV_ATTR_CURSOR_SCROLL_TYPE=>PDO::SQLSRV_CURSOR_BUFFERED));
-	fwrite($noteID, '1');
-	rewind($noteID);
-	$stmt->bindParam(':Name',$noteID,PDO::PARAM_BOOL);
-	$stmt->bindParam(':value',$data);
-	$stmt->execute();
-	
-	
-	$query = "SELECT * FROM PDO_AllTypes WHERE [BigIntCol]=:value";
-	$stmt = $db->prepare($query,array(PDO::ATTR_CURSOR=>PDO::CURSOR_SCROLL, PDO::SQLSRV_ATTR_CURSOR_SCROLL_TYPE=>PDO::SQLSRV_CURSOR_BUFFERED));
-	$stmt->bindParam(':value',$data,PDO::PARAM_BOOL);
-	$stmt->execute();
-	$result = $stmt->fetchColumn(2);	// Retrieve BitCol
-	$result = str_replace("\0","",$result);
-	print("$result\n");
-	
+    // Binding BOOL with buffered queries activated.
+    $noteID = fopen('php://memory', 'a');
+    $data = '0';
+    $query = "UPDATE $tbname SET [BitCol]=:Name WHERE [BigIntCol]=:value";
+    $stmt = $db->prepare($query, array(PDO::ATTR_CURSOR=>PDO::CURSOR_SCROLL, PDO::SQLSRV_ATTR_CURSOR_SCROLL_TYPE=>PDO::SQLSRV_CURSOR_BUFFERED));
+    fwrite($noteID, '1');
+    rewind($noteID);
+    $stmt->bindParam(':Name', $noteID, PDO::PARAM_BOOL);
+    $stmt->bindParam(':value', $data);
+    $stmt->execute();
+
+    $query = "SELECT * FROM $tbname WHERE [BigIntCol]=:value";
+    $stmt = $db->prepare($query, array(PDO::ATTR_CURSOR=>PDO::CURSOR_SCROLL, PDO::SQLSRV_ATTR_CURSOR_SCROLL_TYPE=>PDO::SQLSRV_CURSOR_BUFFERED));
+    $stmt->bindParam(':value', $data, PDO::PARAM_BOOL);
+    $stmt->execute();
+    $result = $stmt->fetchColumn(2);    // Retrieve BitCol
+    unset($stmt);
+    $result = str_replace("\0", "", $result);
+    print("$result\n");
 }
 
-function delete($db)
+function delete($db, $tbname)
 {
-	$data = "STRINGCOL2";
-	$query = "DELETE FROM PDO_AllTypes WHERE [CharCol]=:value";
-	$stmt = $db->prepare($query,array(PDO::ATTR_CURSOR=>PDO::CURSOR_SCROLL, PDO::SQLSRV_ATTR_CURSOR_SCROLL_TYPE=>PDO::SQLSRV_CURSOR_BUFFERED));
-	$stmt->bindParam(':value',$data);
-	$stmt->execute();
-	
+    $data = "STRINGCOL2";
+    $query = "DELETE FROM $tbname WHERE [CharCol]=:value";
+    $stmt = $db->prepare($query, array(PDO::ATTR_CURSOR=>PDO::CURSOR_SCROLL, PDO::SQLSRV_ATTR_CURSOR_SCROLL_TYPE=>PDO::SQLSRV_CURSOR_BUFFERED));
+    $stmt->bindParam(':value', $data);
+    $stmt->execute();
+    unset($stmt);
 }
 
 try {
-
-   $db = connect();
-   insert($db);
-   bindParam_LOB($db);
-   bindParam_STR($db);
-   bindParam_NULL($db);
-   bindParam_INT($db);
-   bindParam_BOOL($db);
-   delete($db);
-   
-   echo "Test Completed";
-}
-
-catch( PDOException $e ) {
-    var_dump( $e );
+    $db = connect();
+    $tbname = "PDO_AllTypes";
+    createAndInsertTableAllTypes($db, $tbname);
+    insert($db, $tbname);
+    bindPARAM_LOB($db, $tbname);
+    bindPARAM_STR($db, $tbname);
+    bindPARAM_NULL($db, $tbname);
+    bindPARAM_INT($db, $tbname);
+    bindPARAM_BOOL($db, $tbname);
+    delete($db, $tbname);
+
```

Comment:
Consider not to capitalized words in the function names except LOB (an acronym). Also, please remove the underscores

Reply:
I left it because the constant is PDO::PARAM_LOB

Reply:
Perhaps only remove the underscores (php coding standard)?